CONTRIBUTING.md

```diff
@@ -323,7 +332,77 @@ Update the version's release date and status from the
 [Manage Versions](https://jira.mongodb.org/plugins/servlet/project-config/PHPC/versions)
 page.
 
-### Update version info
+### Trigger the release workflow
+
+Releases are done automatically through a GitHub Action. Visit the corresponding
+[Release New Version](https://github.com/mongodb/mongo-php-driver/actions/workflows/release.yml)
+workflow page to trigger a new build. Select the correct branch (e.g. `v1.18`)
+and trigger a new run using the "Run workflow" button. In the following prompt,
+enter the version number and the corresponding JIRA version ID for the release.
+This version ID can be obtained from a link in the "Version" column on the
+[PHPC releases page](https://jira.mongodb.org/projects/PHPC?selectedItem=com.atlassian.jira.jira-projects-plugin%3Arelease-page&status=unreleased).
+
+The automation will then create and push the necessary commits and tag, create a
+draft release, and trigger the packaging builds for the newly created tag. The
+release is created in a draft state and can be published once the release notes
+have been updated.
+
+Alternatively, you may follow the [manual release process](#manual-release-process)
+before continuing with the next section.
+
+### Upload package to PECL
+
+Once the packaging workflow has finished creating the PECL package, it will be
+published as a build artifact of the package workflow, as well as a release
+asset in the draft release. This package may be published via the
+[Release Upload](https://pecl.php.net/release-upload.php) form. You will have one chance to confirm the package
+information after uploading.
+
+> **Note:** If downloading the package from the build artifacts, be aware that
```

Comment:
If you'd like to use the special syntax (learned about this while specs were being ported from RST to Markdown): https://github.com/orgs/community/discussions/16925

Reply:
Ah, good point :)